### ISSUE TITLE
'hifi' version of website (272)

### ISSUE THREAD

**Issue by GalacticEmperor1** [pygame-community/pygame-ce#287]
#  **Issue &numero;272 opened by *[illume](https://github.com/illume)* at 2015-08-30 04:12:57**

Originally reported by: **micah page (Bitbucket: [metulburr](https://bitbucket.org/metulburr), GitHub: [metulburr](https://github.com/metulburr))**

----------------------------------------

I understand that you guys want to try something new. I understand that there are already a number of complaints (whether constructive or not) about the website. I even like some additions to the website. But i cannot sit by and not say anything.

1) First of all there should be communication on all networks. For example some people only use reddit/pygame for discussion and help, and use the pygame website for docs. In this case, those people will never see mailinglist threads. I used pygame for 2 years before i even knew there even was a mailing list. Pygame is large enough to have its own forum as its base of communication. Since it does not, its all over the place. Mailing lists, reddit, google groups, facebook, twitter, IRC,  etc. (and who knows where else) The average John Doe just starting python/pygame has to learn to tap into numerous connections to get help for pygame. This alone is daunting. (It would seem more natural to go to the pygame website and sign into a forum.) And all of these connections need to hear from "the almighty" (anyone pushing updates to pygame, and anyone in charge of the main website). Maybe if the community knew what you planned, it would make more sense, and you would get less complaining and more constructive criticism. 
 
tl;dr It would be nice to hear from admins on all networks regarding pygame about major events...especially the main website changing. As well as responding and participating in exchange of ideas regarding the main web site from all networks on a regular basis. 

2) All of my ideas regarding the change of the new site have already been spoken. However, there is not much response to them. Issues are closed, responses ignored, etc. It makes it look like you guys do not care about what the community thinks of the new site. And its a long line of history repeating itself too. For years, new users could not (and still not) sign up to post their projects). A lot of them had no idea where to post (a pygame forum would of been nice) to be allowed to create an account. A lot of them just abandoned the idea of sharing their projects with the "main website". Or worse went to another library. The few, were able to figure out how to create an account. This resonates the first argument. No communication (or very little). 

I am going to withhold my comments about the new site as i fear they will fall on deaf ears. 

----------------------------------------
- Bitbucket: https://bitbucket.org/pygame/pygame/issue/272
<hr>

#  Comments

# # #  **[illume](https://github.com/illume)* commented at 2015-08-30 16:58:39*

*Original comment by* **René Dudfield (Bitbucket: [illume](https://bitbucket.org/illume), GitHub: [illume](https://github.com/illume))**:

----------------------------------------

Hi,

firstly thanks for your response, you make some good points. 

I will respond to all your points, but keep in mind I am just one person, and am a very busy volunteer.

Yes, we should communicate on most networks. The website and previous two sprints have been mentioned on at least twitter, reddit, facebook, the website, and the mailing list.

I don't know which of your issues have been closed, or which of your emails haven't been answered... I can't find them. If you can point me to them I will do my best.

Signing up to be able to post projects, rather than require people to engage with the community to get an account is something on the list of things I want to do in the near term. This is the main group of people I want to help out -- the people who make things, and want to share projects. It's important to me. I noticed you've got a few projects on bitbucket (when I was trying to find your issues), and see that you're one of the programmer types. So I am especially interested in ideas you may have regarding sharing projects. I also hope you understand how hard it is to work on open source projects in your spare time.

At this point in time, I have a long list of features for the website planned, and whilst comments on it are important I think there is enough work to do for now. I've invited people to participate in sprints for the website, and I hope perhaps at least one other person will get involved and actually help out at the next sprint. Ideas are cheap, making things is harder. This gets back to communication, and regular sprints communicated better I hope will reinvigorate the community.

Luckily there are many respectful people in the pygame community -- the makers, the doers, the coders, the artists mostly. I especially have time for people prepared to roll up their sleeves and help out, or who want to work on their projects and share them.

I appreciate how you've been respectful in your response, and have made a few really good points. Hopefully you can rest safe in knowing that 1) communication is beginning to improve, and that 2) new projects are going to be easier to add ( https://bitbucket.org/pygame/pygame/issues/262/share-work-add-new-project ). Also I wanted to point out that I'll put the most recent mailing list posts on the website too, so the mailing list is more obvious ( # 274 ).

Happy hacking!

<hr>

# # #  **[illume](https://github.com/illume)* commented at 2015-08-30 22:15:02*

*Original comment by* **micah page (Bitbucket: [metulburr](https://bitbucket.org/metulburr), GitHub: [metulburr](https://github.com/metulburr))**:

----------------------------------------

I am one of those that dont use all the possible avenues of social media for pygame. For pygame...i do not use facebook, mailing threads, twitter, etc. I only use reddit/pygame. There are a lot of annoyances of cross posting a question among numerous methods. It is much easier and quicker to get a response from reddit. 

I use github the majority of the time. I really only use bitbucket when i want a repo secret. Which is not often. 
https://github.com/metulburr

It is not my issues/posts closed. It was others. Though their comments might have crossed the line of being cruel, their comments were in tune with the rest of the community in regards to the new site.

I do understand working on open source projects in your spare time as i do that as well. I do understand it takes time. 

I understand that the new site is entangled with the old site. So it makes sense that links are broken, columns not completed, etc. 

Now onto the site...Shouldn't the download page, documentation, and about pygame be the number one priority for the first few seconds of seeing the main site? As someone experienced in using many different libraries and different languages, I often navigate to these 3 things first. When i first opened the new site, it took me a long time to find these 3. 

What prevented me in finding these sooner was: 1) the scrolling side to side. No other site uses this so i had to learn to use it 2) the category names. I would assume there should naturally be a download one leading to (temporarily) to the old site. "Make" and "create" are synonyms, but yet only one has the download link. I still have to click both to find the link even after already finding it yesterday.

I have to click through to find the documentation column as i am not sure where it is. "Learn" could be, but so could other tabsa. 

I fear new pygamers coming too. That they may not be able to navigate the site and just get frustrated with it. Coding is frustrating enough, let alone finding the important 3 (download, about, and docs). I know what to look for too. New coders do not always. 

I feel like this site should be on the back burner like it was the past 6 months until it is mostly functional and filled out. 

---

All of these points collide with communication too. As an every day pygame user, i dont know if you plan on reverting some of this stuff. Is it a temp place holder? Is it suppose to be like this? What exactly do you envision as the end result?  

<hr>

# # #  **[illume](https://github.com/illume)* commented at 2015-08-31 06:53:21*

*Original comment by* **Horst JENS (Bitbucket: [horstjens](https://bitbucket.org/horstjens), GitHub: [horstjens](https://github.com/horstjens))**:

----------------------------------------

just want to chip in that i use pygame for years (i still own http://thepythongamebook.com ) and i'm very happy to see activity and a new look at the front site. Pygame is alive! Will try to get more active via bitbucket.

<hr>

# # #  **[illume](https://github.com/illume)* commented at 2015-08-31 12:38:01*

*Original comment by* **micah page (Bitbucket: [metulburr](https://bitbucket.org/metulburr), GitHub: [metulburr](https://github.com/metulburr))**:

----------------------------------------

pygame has always been alive. The old website just failed to show it by not updating download links, news rarely updated, new accounts frozen with a secret link to bypass spam,  activity of new pygame versions not linked in the downloads page (making it look like pygame was abandoned), examples not updated, etc.

@horstjens Doesnt the horizontal scrolling bother you? Or the naming conventions of the categories? Doesnt it feel hard to find information? (assuming you were not able to go back to the old site)

I do like some things of the new site. Such as a twitter, commits, issues, etc. But that is probably because we have gone years from the old site without updating downloads. Everyone always asked is pygame dead, because there was no real visible way to see the activity of bitbucket from the main site. Everyone sees the last pygame as 2009 and assumed it died. However i dont feel like this should have these should have as much real estate on the site as downloads, docs, and about for first time viewing. 

<hr>

# # #  **[illume](https://github.com/illume)* commented at 2015-08-31 13:57:45*

*Original comment by* **Horst JENS (Bitbucket: [horstjens](https://bitbucket.org/horstjens), GitHub: [horstjens](https://github.com/horstjens))**:

----------------------------------------

@metulburr i'm sure the new site layout can be improved over time. i had no trouble finding the old yellow-green "documentation" and the cookbook both were just one or two clicks away. I rarely used other parts of the old site. As far as i understand the new layout is also on bitbucket and can be improved by pull requests.

<hr>

# # #  **[illume](https://github.com/illume)* commented at 2015-08-31 21:35:34*

*Original comment by* **micah page (Bitbucket: [metulburr](https://bitbucket.org/metulburr), GitHub: [metulburr](https://github.com/metulburr))**:

----------------------------------------

@horstjens Well now that i know where to find things and how to navigate the new site, I can find things. I am now concerned about new pygamers that will go through the dreaded first time view and trying to navigate through the site. I didnt use much from the old site either, but that was because it was outdated. 

Where is the repo for the website? I cannot find it. On the new site the issues just leads back to pygame repo, but i do not see any source for the website. And an issue create says to make the repo public...which apparently it is not? 

I foresee when the public gets a chance to make pull requests on the website, there will be a massive bombardment of such things described above. They do not even have to know Javascript to make one pull request. Naming conventions changed. show/learn/make/create/collaborate/awesome -> About/Download/Documentation/Contribute/Community/News

<hr>

# # #  **[illume](https://github.com/illume)* commented at 2015-09-04 06:53:21*

*Original comment by* **René Dudfield (Bitbucket: [illume](https://bitbucket.org/illume), GitHub: [illume](https://github.com/illume))**:

----------------------------------------

@metulburr I'm closing this issue now. I've linked to other issues where your various points are covered. If I've missed something please feel free to open other issues with one issue per bitbucket issue.

You've called me lazy, and made stuff up about ignoring your emails and issues. I don't expect an apology from you, but saying stuff like this to me is not appreciated. Can you please stop that? Feel free to criticize my work, but personal attacks and libel are not things any human should have to put up with.

<hr>

# # #  **[illume](https://github.com/illume)* commented at 2015-09-04 13:26:12*

*Original comment by* **micah page (Bitbucket: [metulburr](https://bitbucket.org/metulburr), GitHub: [metulburr](https://github.com/metulburr))**:

----------------------------------------

The main factor in laziness I see is the disabling of creating new accounts to avoid removing spam accounts. For a couple months, fine, but it has been at least 3-4 years, i think that it has been disabled.
